src/app/debug/filter-side-drawer/filter-side-drawer.component.ts

```diff
@@ -100,11 +100,10 @@ export class FilterSideDrawerComponent implements OnDestroy, OnInit {
     this.filterService.updateFilterContext(metadataName, '');
   }
 
-  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-  getTooltipSuggestion(key: string) {
+  getTooltipSuggestion(key: string): Partial<Report> | Checkpoint[] | string | number | boolean | undefined {
     if (this.toolTipSuggestions) {
       return this.toolTipSuggestions[key as keyof Report];
     }
-    return null;
+    return undefined;
   }
```

Comment:
Sorry Martijn, I gave the wrong suggestion earlier because we are not returning an object with one of the keys, but just the key. The code should be like this:
```typescript
getTooltipSuggestion<K extends keyof Report>(key: K): Report[K] | undefined {
  return this.toolTipSuggestions?.[key];
}
```
It is also a bit cleaner now that it returns `undefined`.